cmd/ctr/commands/namespaces/namespaces.go

```diff
@@ -174,10 +174,13 @@ var removeCommand = &cli.Command{
 					log.G(ctx).WithError(err).Errorf("unable to delete %v", target)
 					continue
 				}
-
+				if errdefs.IsNotFound(err) {
```

Comment:
Can this be the `else` part of the condition?.  

Or we can first check for `IsNotFound` err.

Reply:
thanks for reviewing. updated.